grails-app/services/org/pih/warehouse/stockTransfer/StockTransferService.groovy

```diff
@@ -194,7 +195,7 @@ class StockTransferService {
         }
         order.save(failOnError: true)
 
-        def currentLocation = authService.currentLocation
+        def currentLocation = AuthService?.currentLocation
```

Comment:
Instead of replacing it with `AuthService`, please just inject service dependency as `def authService` at the beginning of this service class.